### ISSUE TITLE
Using `C#` built-in `Timer` Crashes Android `AAB` Export

### ISSUE THREAD

**Issue by MrZak-dev** [godotengine/godot#55638]
### Godot version

3.4.mono.stable , 3.4.1.mono.rc1

### System information

Android

### Issue description

Using `C#` built-in `Timer` Crashes Android `AAB` Export.

maybe releated to https://github.com/godotengine/godot/issues/48458 , 

### Steps to reproduce

- New Project 
- Use C# built-in `Timer`
```csharp
public override void _Ready()
    {
        MyTimer = new Timer();
        MyTimer.Interval = 1000;
        MyTimer.Elapsed += (sender, args) => {
            GD.Print("Hello");
        };
        MyTimer.Start();
        
    }
```

- Export game to `AAB`
- Extract APKs from exported `AAB` using [Bundletool](https://developer.android.com/studio/command-line/bundletool)
```powershell
$~ java -jar bundletool-all-1.8.2.jar build-apks --bundle=./exported.aab --output=./output.apks
```
- Install `output.apks` to a connected Android device
```powershell
$~ java -jar bundletool-all-1.8.2.jar install-apks --apks .\output.apks
```

Crash log 
```log
--------- beginning of crash
2021-12-01 13:13:00.942 7285-7398/? A/libc: FORTIFY: pthread_mutex_lock called on a destroyed mutex (0xc17ed80c)
2021-12-01 13:13:00.946 7285-7398/? A/libc: Fatal signal 6 (SIGABRT), code -1 (SI_QUEUE) in tid 7398 (Thread Pool Wor), pid 7285 (icopo.SpaceFire)
2021-12-01 13:13:00.939 7285-7342/? E/mono: Unhandled Exception:
    System.DllNotFoundException: __Internal assembly:<unknown assembly> type:<unknown type> member:(null)
        at (wrapper managed-to-native) System.TimeZoneInfo+AndroidTimeZones.monodroid_get_system_property(string,intptr&)
      at System.TimeZoneInfo+AndroidTimeZones.GetDefaultTimeZoneName () [0x00029] in <a947100bfc9e4b689622af54e1e07934>:0 
      at System.TimeZoneInfo+AndroidTimeZones.get_Local () [0x00000] in <a947100bfc9e4b689622af54e1e07934>:0 
      at System.TimeZoneInfo.CreateLocal () [0x00000] in <a947100bfc9e4b689622af54e1e07934>:0 
      at System.TimeZoneInfo.get_Local () [0x00009] in <a947100bfc9e4b689622af54e1e07934>:0 
      at System.TimeZoneInfo.GetDateTimeNowUtcOffsetFromUtc (System.DateTime time, System.Boolean& isAmbiguousLocalDst) [0x00000] in <a947100bfc9e4b689622af54e1e07934>:0 
      at System.DateTime.get_Now () [0x00008] in <a947100bfc9e4b689622af54e1e07934>:0 
      at System.Timers.Timer.MyTimerCallback (System.Object state) [0x00019] in <ff8dc755b313498597c01f2c3444935b>:0 
      at System.Threading.Timer+Scheduler.TimerCB (System.Object o) [0x00007] in <a947100bfc9e4b689622af54e1e07934>:0 
      at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem () [0x00008] in <a947100bfc9e4b689622af54e1e07934>:0 
      at System.Threading.ThreadPoolWorkQueue.Dispatch () [0x00074] in <a947100bfc9e4b689622af54e1e07934>:0 
      at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback () [0x00000] in <a947100bfc9e4b689622af54e1e07934>:0 
```

### Minimal reproduction project

[UnisgTimersAABCrash.zip](https://github.com/godotengine/godot/files/7656241/UnisgTimersAABCrash.zip)

**Comment by Calinou** [comment]
@MrZak-dev Does it crash if you export an APK instead?

**Comment by MrZak-dev** [comment]
@Calinou No it doesn't , it crashed only when you export APK and install it directly from Google play or using Bundletool and exporting apks from The `AAB`

**Comment by oknpk** [comment]
Hi, is there any solution? 
After uploading the app to google play, the app crashes after splash screen. 
No problem as .APK, now I'm trying to remove the timers and datetime methods from the project.

**Comment by MrZak-dev** [comment]
the only solution i found for my particular project is to not use C# built-in timers and i have switched all of them to the regular Godot timer Nodes .

**Comment by oknpk** [comment]
After a little look, my problem was different. I am using LiteDB database and LINQ. These also crash. I'll open a new issue after doing some more research.

**Comment by akien-mga** [comment]
Fixed by #57420.
